src/class/bth/bth_device.h

```diff
@@ -39,6 +39,13 @@
 #ifndef CFG_TUD_BTH_DATA_EPSIZE
 #define CFG_TUD_BTH_DATA_EPSIZE      64
 #endif
+#ifndef CFG_TUD_BTH_HISTORICAL_COMPAT
+#define CFG_TUD_BTH_HISTORICAL_COMPAT 0
+#endif
+#ifndef CFG_TUD_BTH_ISO_ALT_COUNT
+// See Bluetooth Core v5.3, Vol. 4, Part B, Section 2.1
+#error You must tell the driver the number of ISO endpoints to use
```

Comment:
this check does not do much since CFG_TUD_BTH_ISO_ALT_COUNT is default to 0 here https://github.com/hathach/tinyusb/blob/master/src/device/usbd.h#L774

we should move this to tusb_opt.h and only force it when `CFG_TUD_BTH` is enabled.